clients/datasource/internal/pypi/pypi.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pypi defines the structures to parse PyPI JSON API response.
+package pypi
+
+// Response defines the response of PyPI JSON API.
+type Response struct {
+	Info     Info     `json:"info"`
+	Releases Releases `json:"releases"`
```

Comment:
hmm looking at https://docs.pypi.org/api/json/#get-a-project:

> **Deprecated keys**
> 
> The following keys are considered deprecated:
> - releases: projects should shift to using the [Index API](https://docs.pypi.org/api/index-api/) to get this information, where possible.
>
> ...
>
> In the future, each of these keys may be removed entirely from this API response.

The Index API looks annoying to parse, though.